src/components/Login.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+export default function Login() {
+  interface User {
```

Comment:
1. interface should be top-level construct
2. can we qualify the interface name? `UserCredentialsInput` or smth like that. I foresee we will use `User` for smth else heh

Reply:
1. Shifted all interfaces to be top-level construct.
2. Qualified all the interface names to be practical

Reply:
Please push your changes :)